lib/application-processing/resolvers.ts

```diff
@@ -947,6 +947,20 @@ export const updateApplicationProcessingUploadDocuments: Resolver<
     throw new ApolloError('Not authenticated');
   }
 
+  if (!process.env.APPLICATION_DOCUMENT_LINK_TTL_DAYS) {
+    throw new ApolloError('Application document link duration not defined');
+  }
+
+  let signedUrl;
+  try {
+    const durationSeconds = parseInt(process.env.APPLICATION_DOCUMENT_LINK_TTL_DAYS) * 24 * 60 * 60;
+    if (documentsUrl) {
+      signedUrl = getSignedUrlForS3(documentsUrl, durationSeconds);
+    }
+  } catch (e) {
+    throw new ApolloError(`Error uploading application document to AWS: ${e}`);
+  }
+
```

Comment:
If we're generating the signed URL on every query request, then we don't need to save it in the db, so these changes can be moved from here to `lib/applications/field-resolvers.ts`